src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -78,53 +89,119 @@ public void Start(ApplicationData applicationData)
 
             // If we have PassThru, then we want to make them selectable. If we make them selectable,
             // they have a 8 character addition of a checkbox ("     [ ]") that we have to factor in.
-            int offset = applicationData.PassThru ? 8 : 4;
+            _listViewOffset = _applicationData.PassThru ? 8 : 4;
```

Comment:
It'd be cool if this could be pulled up to the top of `Start` since it's a glorified "constant" for the execution... But I won't block on this.

Reply:
can move